src/RiakClient/Models/RiakPutOptions.cs

```diff
@@ -125,24 +77,8 @@ internal void Populate(RpbPutReq request)
 
             if (Timeout.HasValue)
             {
-                request.timeout = Timeout.Value;
+                request.timeout = (uint)Timeout.Value;
```

Comment:
Let me put in a check that Timeout never be negative, as this will cast it to a huge number.